container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -1883,34 +1871,22 @@ configuration:
     description: "log level: debug, info, error, fatal"
   - name: AUTOSCALER_INSTANCE_METRICS_DB_CONNECTION_CONFIG_CONNECTION_MAX_LIFETIME
     default: "60s"
-    description: "The maximum lifetime of instance metrics database connection in connection pool"
+    description: "The maximum age of a connection that can be reused for the Autoscaler Instance_Metrics database."
   - name: AUTOSCALER_INSTANCE_METRICS_DB_CONNECTION_CONFIG_MAX_IDLE_CONNECTIONS
     default: 10
-    description: "The maximum idle instance metrics database connections in connection pool"
+    description: "The maximum number of idle connections to the Autoscaler Instance_Metrics database."
   - name: AUTOSCALER_INSTANCE_METRICS_DB_CONNECTION_CONFIG_MAX_OPEN_CONNECTIONS
     default: 100
-    description: "The maximum number of instance metrics database connections in connection pool"
-  - name: AUTOSCALER_INSTANCE_METRICS_DB_PORT
-    default: 5432
-    description: "The autoscaler instance metrics database server port"
-  - name: AUTOSCALER_INSTANCE_METRICS_DB_SCHEME
-    default: "postgres"
-    description: "The autoscaler instance metrics database scheme"
+    description: "The maximum number of connections to the Autoscaler Instance_Metrics database."
   - name: AUTOSCALER_LOCK_DB_CONNECTION_CONFIG_CONNECTION_MAX_LIFETIME
     default: "60s"
-    description: "The maximum lifetime of lock database connection in connection pool"
+    description: "The maximum age of a connection that can be reused for the Autoscaler lock database."
   - name: AUTOSCALER_LOCK_DB_CONNECTION_CONFIG_MAX_IDLE_CONNECTIONS
     default: 10
-    description: "The maximum idle lock database connections in connection pool"
+    description: "The maximum number of idle connections to the Autoscaler lock database."
   - name: AUTOSCALER_LOCK_DB_CONNECTION_CONFIG_MAX_OPEN_CONNECTIONS
     default: 100
-    description: "The maximum number of lock database connections in connection pool"
-  - name: AUTOSCALER_LOCK_DB_PORT
-    default: 5432
-    description: "The autoscaler lock database server port"
-  - name: AUTOSCALER_LOCK_DB_SCHEME
-    default: "postgres"
-    description: "The autoscaler lock database scheme"
+    description: "The maximum number of connections to the Autoscaler lock database."
   - name: AUTOSCALER_METRICS_COLLECTOR_COLLECTOR_COLLECT_INTERVAL
     default: "30s"
     description: "The duration defines how long does metricscollector aggregate collected instance metrics"
```

Comment:
`AUTOSCALER_METRICS_COLLECTOR_LOGGIN_LEVEL` must be removed in favor of `LOG_LEVEL`.